ibridges/__main__.py

```diff
@@ -32,6 +32,10 @@
     sync:
         Synchronize files/folders and data objects/collections with each other.
         Only updated files will be downloaded/uploaded.
+    list:
+        List the content of a collections, if no path is given, the home collection will be listed.
+    midir:
```

Comment:
Shouldn't this be `mkcol`?

Reply:
Yes and no ... I am a bit torn, for people who have experience with icommands (imkdir) or bash mkdir might be more intuitive ... to be fully RDM compatible we should switch to mkcoll or something like this.
Any preference?